Adds Twill keyboard (#9661)
qmk/qmk_firmware: keyboards/twill/readme.md

# Twill

> A 60% keyboard with only through hole components and RGB underglow.   

Reviewer: ```suggestion
A 60% keyboard with only through hole components and RGB underglow.
```

Based off the Gingham designed by [Yiancar](https://yiancar-designs.com/portfolio/gingham/)

> ![Twill image](https://imgur.com/gallery/gNAEsL6)

Reviewer: ```suggestion
![Twill image](https://i.imgur.com/s4UtWqF.jpg)
```

> * Keyboard Maintainer: [UWA Makers](https://wiki.uwamakers.com//)  

Reviewer: ```suggestion
* Keyboard Maintainer: [UWA Makers](https://wiki.uwamakers.com/)
```

> * Hardware Supported: Twill PCB rev1 with ATMEGA328p 

Reviewer: ```suggestion
* Hardware Supported: Twill PCB rev1 with ATMEGA328p
```

> Make example for this keyboard (after setting up your build environment):
>     qmk compile -kb twill
> 
> Flash firmware:
>     // In bootloader mode
>     qmk flash -kb twill

Reviewer: ```suggestion
Make example for this keyboard (after setting up your build environment):

    make twill:default

Flashing example for this keyboard:

    make twill:default:flash
```

# Twill

> A 60% keyboard with only through hole components and RGB underglow.
> 
> Based off the Gingham designed by [Yiancar](https://yiancar-designs.com/portfolio/gingham/)
> 
> ![Twill image](https://i.imgur.com/s4UtWqF.jpg)

Reviewer 2: ```suggestion
![Twill image](https://i.imgur.com/s4UtWqFl.jpg)

A 60% keyboard with only through hole components and RGB underglow. Based on the Gingham designed by [Yiancar](https://yiancar-designs.com/portfolio/gingham/).
```
